Automattic/jetpack
# Boost: Fix ISA summary UI not showing up

Fixes https://github.com/Automattic/boost-cloud/issues/376

## Proposed changes:

* Fixes the summary UI not showing up after requesting a report and refreshing the page.

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Request an ISA report;
* Refresh the page and make sure the Summary UI shows up properly
![image](https://github.com/Automattic/jetpack/assets/11799079/c80df7e2-7693-4d98-9e35-7a8171824f91)
